src/screens/TopicPageScreen/index.js

```diff
@@ -448,8 +415,12 @@ const TopicPageScreen = (props) => {
         onRefresh={onRefresh}
         refreshing={loading}
         renderItem={renderItem}
-        onScroll={handleScrollEvent}
-        onScrollBeginDrag={handleOnScrollBeginDrag}
+        onScroll={handleScroll}
+        scrollEventThrottle={16}
+        showSearchBar={true}
+        searchHeight={headerShowHeight}
+        contentOffset={{x: 0, y: topPosition}}
+        contentInsetAdjustmentBehavior={feeds?.length > 1 ? 'automatic' : 'never'}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onScrollBeginDrag` prop has been removed and the `onScroll` prop has been changed. Make sure this doesn't affect the scroll behavior of the component.
```diff
-        onScroll={handleScrollEvent}
-        onScrollBeginDrag={handleOnScrollBeginDrag}
+        onScroll={handleScroll}
+        scrollEventThrottle={16}
+        showSearchBar={true}
+        searchHeight={headerShowHeight}
+        contentOffset={{x: 0, y: topPosition}}
+        contentInsetAdjustmentBehavior={feeds?.length > 1 ? 'automatic' : 'never'}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->